src/main/java/seedu/address/model/event/Time.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.event;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
```

Comment:
Add class-level javadocs to tell us about the class

Reply:
Done-by Sol